packages/plugin-infra/src/manager/site-adaptor.ts

```diff
@@ -34,7 +34,16 @@ useActivatedPluginsSiteAdaptor.visibility = {
 }
 
 export function useIsMinimalMode(pluginID: string) {
-    return useValueRef(minimalModeSub).includes(pluginID)
```

Comment:
why do you need this change? useValueRef already handles `readyPromise`.

<img width="532" alt="" src="https://github.com/DimensionDev/Maskbook/assets/5390719/ffef521e-5ed5-44a5-9a53-13ca051343a3">

Reply:
The result was `false` by default, that's arbitrary.

Reply:
@Jack-Works Any other questions?

Reply:
the `use(ref.readyPromise)` should suspend the render, you should not get a "default" value before the `readyPromise` is resolved.

Reply:
```ts
        if (!ref2.nowReady) use(ref2.readyPromise)
```
Should be 
```ts
        if (!ref2.ready) use(ref2.readyPromise)
```

Reply:
nice catch